stryker4jvm-mutator-kotlin/src/main/kotlin/stryker4jvm/mutator/kotlin/KotlinCollector.kt

```diff
@@ -45,4 +46,8 @@ class KotlinCollector : Collector<KotlinAST> {
 
         return res
     }
+
+    fun collect(tree: KotlinAST?): CollectedMutants<KotlinAST> {
+        return collect(tree, LanguageMutatorConfig(mutableSetOf()))
```

Comment:
If you are not using the languageconfig, can't you leave it null and add a nullable to collect(tree, config)?

Reply:
you are right, I fogot about that because instead of compiler issues I was given a runtime exception because the private config variable of the base language mutator was null

Reply:
has now been changed